torchvision/prototype/datasets/_builtin/dtd.py

```diff
@@ -98,7 +90,7 @@ def _make_datapipe(
 
         splits_dp = Filter(splits_dp, path_comparator("name", f"{config.split}{config.fold}.txt"))
         splits_dp = LineReader(splits_dp, decode=True, return_path=False)
-        splits_dp = Shuffler(splits_dp, buffer_size=INFINITE_BUFFER_SIZE)
+        splits_dp = hint_shuffling(splits_dp)
```

Comment:
This was unintentionally already correct. This was only detected, because the test did not fail without me fixing the dataset. Things like this will be prevented in the future.